daskms/experimental/arrow/reads.py

```diff
@@ -228,6 +228,11 @@ def xds_from_parquet(store, columns=None, chunks=None, **kwargs):
     else:
         raise TypeError("chunks must be None or dict or list of dict")
 
+    if store.table:
+        table_root = ""
+    else:
+        table_root = "MAIN"
+
```

Comment:
```suggestion
    table_root = "" if store.table else "MAIN"
```

Also, could we use `table_root` in other places (I've noted them lower down)

Reply:
Done, and I chose to go with `table_path` everywhere.